### ISSUE TITLE
Bucket Update support

### ISSUE THREAD

**Issue by GabriFedi97** [fsouza/fake-gcs-server#1238]
Looks like that the bucket update API is not yet supported. 
From the `docs`  it should be a PUT method on path `/b/{bucketName}`. 

I can try to provide an implementation and contribute to the project if it could speed up the process. 

Google Cloud docs: https://cloud.google.com/storage/docs/json_api/v1/buckets/update

**Comment by fsouza** [comment]
Yeah, right now bucket update isn't supported because there's nothing to update: bucket attributes are not supported yet (they will soon though, see #1174).
